lib/smart_answer_flows/check-uk-visa/outcomes/outcome_transit_to_the_cta.govspeak.erb

```diff
@@ -0,0 +1,33 @@
+<% content_for :title do %>
+  You’ll need a visa to pass through the UK (unless you’re exempt)
+<% end %>
+
+<% content_for :body do %>
+
```

Comment:
Very minor: I don't think we need this blank line or the one at the bottom of the body.